tests/pytorch/test_pytorch_model_export.py

```diff
@@ -738,6 +760,33 @@ def test_load_model_succeeds_when_data_is_model_file_instead_of_directory(
     )
 
 
+@pytest.mark.large
+@pytest.mark.parametrize("scripted_model", [True, False])
+def test_log_state_dict(sequential_model, model_path, data):
+    artifact_path = "pytorch_model"
+    with mlflow.start_run():
+        mlflow.pytorch.log_state_dict(
+            state_dict=sequential_model.state_dict(), artifact_path=artifact_path
+        )
+        model_path = mlflow.get_artifact_uri(artifact_path)
+
+    model_class = nn.Sequential(nn.Linear(4, 3), nn.ReLU(), nn.Linear(3, 1),)
+    model_state_dict = mlflow.pytorch.load_state_dict(model_path)
+    model_class.load_state_dict(model_state_dict)
+    model_class.eval()
+
+    assert (
+        len(set(model_state_dict.items()).intersection(set(sequential_model.state_dict().items())))
+        == 0
+    )
+
+    np.testing.assert_array_almost_equal(
+        pd.DataFrame(_predict(model=model_class, data=data)),
+        pd.DataFrame(_predict(model=sequential_model, data=data)),
```

Comment:
I don't think we need to use `pd.DataFrame`.

Reply:
I have removed it.